### PR TITLE
feat(examples): add protractor angular architect

### PR DESCRIPTION
**Pull request by Toxicable**
Adds required target and configuration to make protractor on the Angular Bazel Architect example work under bazel.

### CODE REVIEW

**Comment by Toxicable** [comment]
failing in circleci with

>[19:55:18] E/launcher - session not created: This version of ChromeDriver only supports Chrome version 79

Not sure what version is installed in circleci

**Comment by gregmagolan** [comment]
Chrome version is whatever comes with circleci/node:10.16-browsers.

You could try updating the images to `circleci/node:12.14-browsers` && `circleci/node:12.14`. We don't rely on the version of node from the images since node_repositories handles download node for bazel

**Comment by gregmagolan** [comment]
Newest images will probably have Chrome 79 or 80.

**Comment by Toxicable** [comment]
Awesome, that did it!

File: examples/angular_bazel_architect/e2e/protractor.conf.js [hunk 1]
```diff
@@ -10,13 +10,21 @@ const {SpecReporter} = require('jasmine-spec-reporter');
...
   onPrepare() {
     require('ts-node').register({project: require('path').join(__dirname, './tsconfig.json')});
     jasmine.getEnv().addReporter(new SpecReporter({spec: {displayStacktrace: true}}));
+  },
+  capabilities: {
+    browserName: 'chrome',
+    pageLoadStrategy: 'normal',
+    chromeOptions: {
+      args: [
+        '--headless', '--disable-gpu', '--window-size=800,600', '--debuggerAddress=127.0.0.1:12633'
```
Comment (gregmagolan) [inline examples/angular_bazel_architect/e2e/protractor.conf.js]: Here's what I used in an angular PR that I was working on that covers most of the issues that can come up with headless and on different CI platforms:
```
args: ['--no-sandbox', '--headless', '--disable-gpu', '--disable-dev-shm-usage', '--hide-scrollbars', '--mute-audio']
```
Why the `--windows-size` and `--debuggerAddress` here? Needed for this to work?

Comment (gregmagolan) [reply examples/angular_bazel_architect/e2e/protractor.conf.js]: Documented where those args came from here: https://github.com/angular/angular/pull/33927/files#diff-303c8e2af10b19172edb0209e673a17b if you're curious

**Comment by gregmagolan** [comment]
@googlebot I consent

File: examples/angular_bazel_architect/e2e/protractor.conf.js (hunk 1 above)
Comment (gregmagolan) [reply examples/angular_bazel_architect/e2e/protractor.conf.js]: Discussed offline & cleaned up.

**Comment by gregmagolan** [comment]
@googlebot I consent.

**Pull request merged**